quiz.js

```diff
@@ -48,7 +82,10 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.update({
+  name: 'Arryn'
+}, { $unset: {motto:}
```

Comment:
Almost! You're just missing one thing. Mongo is a stickler for always having values for its keys, even when a key is about to be destroyed. As it is, this instruction will not run, but if you add a value for `motto` (even an empty string) it will work.
```
db.houses.update({ name : 'Arryn' }, { $unset : { motto : "" } })
```